pynicotine/gtkgui/about.py

```diff
@@ -57,24 +62,22 @@ def __init__(self, parent, nicotine):
 
         self.nicotine = nicotine
 
-        gtk.Dialog.__init__(self, _("About Nicotine+"), parent, gtk.DIALOG_MODAL)
+        gtk.Dialog.__init__(self, _("About Nicotine+"), parent, gtk.DialogFlags.MODAL)
```

Comment:
@lene, feel free to add yourself as Gtk+ 3 / Python 3 porter or something similar. Your contribution deserves recognition.